core/src/main/java/com/graphhopper/routing/util/parsers/OSMRoadAccessParser.java

```diff
@@ -60,10 +60,32 @@ public IntsRef handleWayTags(IntsRef edgeFlags, ReaderWay readerWay, boolean fer
         SpatialRuleSet spatialRuleSet = readerWay.getTag("spatial_rule_set", null);
         if (spatialRuleSet != null && spatialRuleSet != SpatialRuleSet.EMPTY) {
             RoadClass roadClass = RoadClass.find(readerWay.getTag("highway", ""));
-            accessValue = spatialRuleSet.getAccess(roadClass, TransportationMode.MOTOR_VEHICLE, YES);
+            accessValue = spatialRuleSet.getAccess(roadClass, TransportationMode.CAR, YES);
         }
-            
+
         roadAccessEnc.setEnum(false, edgeFlags, accessValue);
         return edgeFlags;
     }
+
+    public static List<String> toOSMRestrictions(TransportationMode mode) {
+        switch (mode) {
+            case FOOT:
+                return Arrays.asList("foot", "access");
```

Comment:
I'd argue that the restrictions are a defining property of a TransportationMode and would move them to the Enum.

Reply:
But only for OSM data. If you use the transportation mode with anything else, these values might be different?

Reply:
The naming is IMO OSM-specific and should be in the parser

Reply:
> OSM-specific

Good point

Reply:
Also some restrictions could be not existent in the TransportationMode that exist in OSM